research/picai/monai_scripts/readme.md

```diff
@@ -0,0 +1,21 @@
+# Monai Scripts
+
+This directory contains work to integrate the Monai AutoSeg3d pipeline
+
+### AutoSeg3d.py
+
+This file runs the monai autoseg3d pipeline on an nnunet structured dataset.
+
+Autoseg3d is designed to work with the very common [MSD](http://medicaldecathlon.com/) dataset format. The nnunet derives it's dataset format from the MSD guidelines but alters it slightly. One of the main changes is that different modalities/channels are stored as different files. Although nnunet provides a script to convert an MSD dataset into an nnunet dataset, we'd rather not have multiple local copies of the same raw dataset. It's easier to get monai's autoseg3d to work with nnunet datasets than to get nnunet to work with MSD datasets, therefore we choose the nnunet dataset structure as our standard that will work with everything.
+
```

Comment:
" It's easier to get monai's autoseg3d to work with nnunet datasets than to get nnunet to work with MSD datasets, therefore we choose the nnunet dataset structure as our standard that will work with everything." 

I am curious what was actually required to get it to work with files per channel as opposed to a single file. Are we just relying on the loader being able to load sets of file paths into a single tensor or do we have to do anything else? It may be worth a quick mention what we did to support this so people know how it differs from the standard workflow

Reply:
Nice addition!